dev-docs/modules/userid-submodules/lockraim.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: userid
+title: lockr AIM
+description: lockr Alternative Identity Manager sub-module
+useridmodule: lockrAIMIdSystem
+---
+
+Alternative Identity Manager (AIM) is a unified container for identity and data management.
+With AIM’s self-service platform, publishers seamlessly integrate and activate alternative IDs like LiveRamp’s Authenticated Traffic Solution (ATS), Unified ID 2.0 (UID2), ID5 and more. The burden of due diligence and maintenance, coupled with the benefits of server-side calls result in the adoption of multiple alternative IDs, clean rooms like InfoSum and CDPs like Blueconic based on their specific needs.
+
+### **Account Creation | AIM**
+
+Sign up for an [Identity lockr account.](https://sso.loc.kr/console/signup)
+Setup your app and activate the AIM library.
+Compile Prebid with the appropriate configurations, and deploy.
+
+### **Configuration | AIM**
+
+First, make sure to add the lockr’s AIM submodule to your Prebid.js package with:
+The following configuration parameters are available:
+AIM supports all Single Sign On functions, newsletter registrations, UTM parameters, etc. For the sake of clarity, a few examples are shared below.
+
+**Google oAuth:**
+If you are using Google oAuth (_as an example_), the onSignIn function will subsequently call window.lockr.setAdditionalData function and include a raw email.
+
+```javascript
+function onSignIn(googleUser) {
+   pbjs.setConfig({
+      userSync: {
+         userIds: [{
+            name: 'lockrAIMId',
+            params: {
+               email: 'john@example.com',
+               appID: 'e84afc5f-4adf-4144-949f-1de5bd151fcc'
+            }
+         }]
+      }
+    });
+}
+```
+
+**Facebook oAuth:**
+If you are using Facebook Login (_as an example_), the statusChangeCallback function will subsequently call window.lockr.setAdditionalData function and include a raw email.
+
+```javascript
+function statusChangeCallback(response) {
+   console.log('statusChangeCallback');
+   console.log(response);
+   if(response.status === 'connected'){
+     pbjs.setConfig({
+        userSync: {
+           userIds: [{
+              name: 'lockrAIMId',
+              params: {
+                 email: 'john@example.com',
+                 appID: 'e84afc5f-4adf-4144-949f-1de5bd151fcc'
+              }
+           }]
+        }
+      });
+   }else{
+     document.getElementById('status').innerHTML = 'Please login';
+   }
+}
+```
+
+**Note:** The above code can be triggered from anywhere on the domain (i.e. a subscription form).
```

Comment:
Seems like this parameters table ought to be moved up above? It's odd to have params in the middle of examples.